person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age || @parent_permission == true
```

Comment:
- [ ] I believe this should be `of_age?`, (with the question mark)

- [ ] `@parent_permission == true` is unnecessary. By definition, `@parent_permission` is either `true` or `false`, so we can remove the equality sign
  
  We should have something like so:
  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```